### ISSUE TITLE
GNURadio Build unable to locate "sattools.asm_golay_decoder"

### ISSUE THREAD

**Issue by RogerWebb** [mndza/gr-sattools#5]
I've attempted builds on Raspbian and Ubuntu 19.  Results are the same.

Installed GNURadio via "sudo apt-get install gnuradio"
Downloaded this repository
Create "build" folder inside.
cmake -DENABLE_SATTOOLS_ZEROMQ=OFF ../
make -j4
sudo make install

After doing this I see the ASM+Golay Decoder component in example I'm working with (this was red before the install) but when I build the job and attempt to run I get the following error:

AttributeError: 'module' object has no attribute 'asm_golay_decoder'

I've attempted to delete and replace this component by selecting from "sattools" menu on the right-hand side of the screen and re-entering settings but results are the same.  It appears to be an issue with this  build.

Please advise.

**Comment by RogerWebb** [comment]
![image](https://user-images.githubusercontent.com/798515/74991897-38524080-53fc-11ea-8952-d069e0e2c32b.png)

Added screenshot for reference.

**Comment by mndza** [comment]
First thing to try, do this and restart GNURadio after:
`sudo ldconfig`

If this does not work, could you share your CMake output? Please do this from a clean 'build' directory.

My guess about what is happening is that GNURadio is unable to find the shared object file, and my first guess is that it is related to the library paths.

**Comment by RogerWebb** [comment]
Apologies for the delay.  I was out of town for the weekend.  See attached files.  I was able to successfully compile today after taking some extra steps.  I needed to install SWIG and utilize the -DENABLE_SATTOOLS_ZEROMQ=OFF flag to get it to compile error-free.  Doing a "make clean" did not remove references to previously compiled objects that were missing pieces of the library.  I went in and manually removed all files from the file system that came out "Up to date" during my "make install" and recompiled fresh and got the correct output from the supplied testing WAV file.  

If there were a way to detect the ZEROMQ feature's presence rather than having a flag, or perhaps making it an "opt-in" feature, might help avoid some complication in the future.  I'm not familiar with SWIG or if it's necessary for the operation of the tool, but if it's a required dependency, it would be nice if the build process installed it or if this requirement were added to the install instructions (apologies if I've missed anything).  The issue all boiled down to the install failing due to the lack of ZEROMQ and the flag during the first build and rebuilding with the flag did not remove the previous build.

Thanks for your help on this.  I'm going to close the issue, but feel free to reach out with any questions.
